### PR TITLE
[@tools-website] Add 'transaction' module menu items

### PR DESCRIPTION
**Pull request by KristinaSpasevska**

### CODE REVIEW

File: packages/apps/tools/src/utils/getHref.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { menuData } from '@/constants/side-menu-items';
+
+export const getHref = (pathname: string, itemHref?: string): string => {
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/utils/getHref.ts:3]: Can we add some tests for this as well? 😁

File: packages/apps/tools/src/constants/side-menu-items.ts [hunk 2]
```diff
@@ -17,14 +17,19 @@ export const menuData: ISidebarToolbarItem[] = [
...
+    title: 'Transactions',
+    icon: 'Transition',
+    href: 'transactions',
+    items: [
+      {
+        title: 'Transfer Tracker',
+        href: Routes.CROSS_CHAIN_TRANSFER_TRACKER,
+      },
+      {
+        title: 'X-Chain Transfer Finisher',
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/constants/side-menu-items.ts:30]: Should we just write this out as "Cross Chain" to just be explicit?

File: packages/apps/tools/src/components/Common/Layout/partials/Sidebar/styles.css.ts [hunk 3]
```diff
@@ -48,14 +48,16 @@ export const gridMiniMenuLinkButtonStyle = style([
     background: 'none',
     border: 'none',
     cursor: 'pointer',
-    padding: '$5',
+    // paddingY: '$3',
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/components/Common/Layout/partials/Sidebar/styles.css.ts:51]: Can be removed?

File: packages/apps/tools/src/components/Common/Layout/partials/Sidebar/Toolbar.tsx [hunk 4]
```diff
@@ -74,8 +76,9 @@ export const Toolbar: FC = () => {
           <li key={String(item.title)} className={gridMiniMenuListItemStyle}>
             <MenuButton
               {...item}
-              onClick={() => handleItemClick(index)}
+              onClick={(e) => handleItemClick(index)}
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/components/Common/Layout/partials/Sidebar/Toolbar.tsx:79]: `e` not used?

File: packages/apps/tools/src/components/Common/Layout/partials/Sidebar/Toolbar.tsx [hunk 5]
```diff
@@ -26,6 +27,7 @@ export const Toolbar: FC = () => {
 
   const handleItemClick = (index: number): void => {
     setVisibleLinks(false);
+    console.log(toolbar[index]);
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/components/Common/Layout/partials/Sidebar/Toolbar.tsx:30]: Can be removed?

File: packages/apps/tools/src/components/Common/Layout/partials/Sidebar/MenuButton.tsx [hunk 6]
```diff
@@ -12,15 +13,19 @@ export interface IMenuButtonProps
   extends ButtonHTMLAttributes<HTMLButtonElement> {
   title?: string;
   active?: boolean;
+  href?: string;
   icon: keyof typeof SystemIcon;
   rotateClass?: string;
+  onClick: (event: React.MouseEvent) => void;
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/components/Common/Layout/partials/Sidebar/MenuButton.tsx:19]: Isn't this already part of `ButtonHTMLAttributes`?

File: packages/apps/tools/src/components/Common/Layout/partials/Sidebar/Menu.tsx [hunk 7]
```diff
@@ -44,6 +44,7 @@ export const Menu: FC = () => {
   } = useLayoutContext();
 
   const handleCloseMenu = () => {
+    console.log('Closing menu');
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/components/Common/Layout/partials/Sidebar/Menu.tsx:47]: Can be removed?

File: packages/apps/tools/src/utils/getHref.ts (hunk 1 above)
Comment (sanderlooijenga) [reply packages/apps/tools/src/utils/getHref.ts:3]: Haha beat me to it, good job 👌

File: packages/apps/tools/src/components/Common/Layout/partials/Header/Header.tsx [hunk 8]
```diff
@@ -34,6 +35,7 @@ const Header: FC<IHeaderProps> = () => {
     e: React.MouseEvent<HTMLAnchorElement>,
   ): Promise<void> => {
     e.preventDefault();
+    console.log('e.currentTarget.href', e.currentTarget);
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/components/Common/Layout/partials/Header/Header.tsx:38]: Should we maybe add an eslint rule about `console` statements?

**Review by sanderlooijenga — APPROVED** [review]

File: packages/apps/tools/src/components/Common/Layout/partials/Header/Header.tsx (hunk 8 above)
Comment (KristinaSpasevska) [reply packages/apps/tools/src/components/Common/Layout/partials/Header/Header.tsx:38]: We'll probably need the logs for places like these 
Screenshot 2023-11-28 at 10 31 36

However, we can add some proper logger with all the levels we need

Comment (sanderlooijenga) [reply packages/apps/tools/src/components/Common/Layout/partials/Header/Header.tsx]: Yes, but then in those specific cases, we could just disable the eslint rule for those lines, right?

**Pull request merged**